### DISCUSSION TITLE
[Feature Request] Add a sound mixer

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#2926] [Features Request & Ideas | 特性请求 & 头脑风暴]
Currently there is no place in the system to manage all audio sources from one place, as there is no slider to adjust system sounds. I think users would appreciate something with similar functionality as in Windows.

**Comment by meiyixiang** [comment]
27/5000 
This is a question that we have looked at in the assessment how to achieve or at what stage

**Comment by meiyixiang** [comment]
Hello, we have accepted this requirement and will realize it later. You can pay attention to our update log or follow our version release to update

**Comment by raspin0** [comment]
As a note I wish this was implemented in tray icon area. Windows 11 as well as Gnome currently address this by adding sound mixer applet in the settings, however I believe this approach is much less convenient. 
Windows seems to be experimenting with re-introducing tray approach idea in the insider release. https://youtu.be/ZXChixENLKA

**Comment by BLumia** [comment]
As a workaround, one can install `kmix` and use that as a sound mixer. Once installed, it can be accessed from the system tray :)
